src/app/component/Menu/Menu.component.js

```diff
@@ -0,0 +1,423 @@
+                <h3 block="Menu" elem="PageLink">
+                    <Link
+                      to="/page/about-us"
+                      onClick={ () => this.closeMenu() }
+                      block="Menu"
+                      elem="Link"
+                    >
+                        { __('CONTACTS') }
+                    </Link>
+                </h3>
+                <div block="Menu" elem="Social">
+                    <CmsBlock identifiers={ ['social-links'] } />
+                </div>
+            </div>
+        );
+    }
+
+    renderAdditionalInformation(checkMobile = false) {
+        if (checkMobile && !isMobile.any()) {
+            return null;
+        }
+
+        return (
+            <>
+                { this.renderStoreSwitcher() }
+                { this.renderPromotionCms() }
+            </>
+        );
+    }
+
+    renderSubMenuDesktop(itemList) {
+        if (isMobile.any()) {
+            return null;
+        }
+
+        const { activeMenuItemsStack } = this.state;
+        const childrenArray = getSortedItems(Object.values(itemList));
+        const mods = activeMenuItemsStack.length > 0;
+
+        return (
+            <div
+              block="Menu"
+              elem="SubCategoriesWrapper"
+              mods={ { isVisible: mods } }
+            >
+                <div
+                  block="Menu"
+                  elem="SubCategoriesWrapperInner"
+                >
+                    <div
+                      block="Menu"
+                      elem="SubCategories"
+                    >
+                        { childrenArray.map(item => this.renderSubLevel(item)) }
+                    </div>
+                    { this.renderAdditionalInformation() }
+                </div>
+            </div>
+        );
+    }
+
+    renderTopLevel() {
+        const { menu } = this.props;
+        const categoryArray = Object.values(menu);
+
+        if (!categoryArray.length) {
+            return null;
+        }
+
+        const {
+            0: { children, title: mainCategoriesTitle }
+        } = categoryArray;
+
+        const mainMods = { type: 'main' };
+
+        return (
+            <>
+                <div block="Menu" elem="MainCategories">
+                    <ul
+                      block="Menu"
+                      elem="ItemList"
+                      mods={ mainMods }
+                      aria-label={ mainCategoriesTitle }
+                    >
+                        { this.renderFirstLevel(children, mainMods) }
+                    </ul>
+                    { this.renderAdditionalInformation(true) }
+                </div>
+                { this.renderSubMenuDesktop(children) }
+            </>
+        );
+    }
+
+    renderStoreSwitcher() {
+        return <StoreSwitcher />;
+    }
+
+    render() {
+        return (
+            <div
+              block="Menu"
+              elem="MenuWrapper"
+              onMouseLeave={ () => this.closeMenu() }
```

Comment:
The usage of `() => this.closeMenu()` is redundant. You can directly use `this.closeMenu`. This is related to every call of `() => this.closeMenu()`.